packages/web/src/components/gcds-input/gcds-input.stories.tsx

```diff
@@ -79,45 +79,45 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-input
   input-id="${args.inputId}"
   label="${args.label}"
```

Comment:
Same comment about the label: would you mind marking it as required in the story and setting a default type?